sierra_adapter/sierra_item_merger/src/test/scala/uk/ac/wellcome/platform/sierra_item_merger/locals/DynamoDBLocal.scala

```diff
@@ -0,0 +1,63 @@
+package uk.ac.wellcome.platform.sierra_item_merger.locals
+
+import com.amazonaws.services.dynamodbv2.model._
+import com.gu.scanamo.Scanamo
+import org.scalatest.{BeforeAndAfterEach, Suite}
+import uk.ac.wellcome.models.MergedSierraRecord
+import uk.ac.wellcome.dynamo._
+import uk.ac.wellcome.test.utils.DynamoDBLocalClients
+
+import scala.collection.JavaConversions._
+
+trait DynamoDBLocal extends BeforeAndAfterEach with DynamoDBLocalClients {
+  this: Suite =>
+
+  val tableName = "sierraItemsTable"
+
+  deleteTable()
+
+  private val table: CreateTableResult = createTable()
+
+  val streamArn: String =
```

Comment:
I don’t think this is every used, and then you can get rid of the `table` variable as well.